app/models/katello/gpg_key.rb

```diff
@@ -6,21 +6,32 @@ class GpgKey < Katello::Model
 
     has_many :repositories, :class_name => "Katello::Repository", :inverse_of => :gpg_key, :dependent => :nullify
     has_many :products, :class_name => "Katello::Product", :inverse_of => :gpg_key, :dependent => :nullify
-
+    has_many :sslca_products, :class_name => "Katello::Product", :foreign_key => "ssl_ca_cert_id",
+                              :inverse_of => :ssl_ca_cert, :dependent => :nullify
+    has_many :sslclient_products, :class_name => "Katello::Product", :foreign_key => "ssl_client_cert_id",
+                                  :inverse_of => :ssl_client_cert, :dependent => :nullify
+    has_many :sslkey_products, :class_name => "Katello::Product", :foreign_key => "ssl_client_key_id",
```

Comment:
In all code instances like these, could these be renamed to `ssl_` to help with readability and being consistent across all uses?

Reply:
fixed. Thanks